test/browser/index.js

```diff
@@ -23,25 +23,19 @@ describe('App', () => {
 
   describe('routing', () => {
```

Comment:
I'm with you, I'm not a fan of commenting any code. Motivation here was to leave at least one example for when we do implement some tests. (If we never do them, the commented will in fact remain...) But I will try to get a minimal test working, just to serve as an example for when we do start the tests.